hsc/crawler.py

```diff
@@ -40,11 +41,13 @@ def __init__(self):
 		self.options = {}
 
 	def login(self, username, password):
-		resp = self.session.get(self.login_url, auth=(username, password))
-		self.cookies = self.session.cookies.get_dict()
-		self.headers = resp.request.headers
-		self.get_number_of_submissions()
-		return self.total_submissions != 0
+		resp = self.session.post(self.login_url, auth=(username, password), headers={'user-agent': self.user_agent})
+		data = resp.json()
+		if data['status']:
+			self.cookies = self.session.cookies.get_dict()
+			self.headers = resp.request.headers
+			self.get_number_of_submissions()
+		return data['status']
```

Comment:
Earlier it was `return self.total_submissions != 0`. Will this change to `return data['status']` have some impact?

Reply:
Yes because a user may have 0 submissions. So it's wrong (hack) to judge login based on the count.